### PR TITLE
Fixes CI configuration and make tests passes

### PR DESCRIPTION
**Pull request by pftg**

### CODE REVIEW

**Comment by pftg** [comment]
@jcupitt sorry, have not verified linter CI before the merge. Please check here, we now it should works 
image

**Comment by pftg** [comment]
some tests are failing will fix them soon

**Comment by jcupitt** [comment]
No problem, and thanks again for working on this.

**Comment by pftg** [comment]
@jcupitt when I use libvips 8.9.1 there is warning like `can't set metadata "offset" on shared image` and for truffleruby it does not add the field to the image.

I thought this is only truffleruby problem only, but now I see that for ubuntu 20, libvips does not update the field for ruby 2.7.2 as well. Do you know why it could be?

**Comment by jcupitt** [comment]
Yes, there was a breaking change in the way metadata modification was handled in libvips 8.9, unfortunately :( 8.10 handles this more gracefully.

I should add something to the ruby-vips test suite to skip these tests on that version.

**Comment by jcupitt** [comment]
Off-topic, but I suppose rubocop can be removed now that you've switched it to standardrb, is that right?

**Comment by pftg** [comment]
@jcupitt so if I skip ti for 8.9 version that is expected behavior?

**Comment by pftg** [comment]
@jcupitt I'll remove rubocop in spearate pr, that will be epic PR with all files changes

**Comment by jcupitt** [comment]
Yes, sorry. I'll make a PR that fixes the test suite with 8.9.

**Comment by jcupitt** [comment]
(this is partly why the old travis CI script built a specific version of libvips ... it reduced the complexity from having ruby-vips and libvips versioning decoupled)

**Comment by pftg** [comment]
Cool fixed for ubuntu 20, found that 18 and 16 has some broken test, will handle them in different pr

**Comment by pftg** [comment]
@jcupitt please check and merge, all tests should be green. But I added some version verifications

File: spec/connection_spec.rb [hunk 1]
```diff
@@ -1,170 +1,163 @@
 require 'spec_helper.rb'
 
-if Vips::at_least_libvips?(8, 9)
-  RSpec.describe Vips::Source do
-    it 'can create a source from a descriptor' do
-      source = Vips::Source.new_from_descriptor(0)
+RSpec.describe Vips::Source, version: [8, 9] do
```
Comment (jcupitt) [inline spec/connection_spec.rb:3]: Oh nice, I didn't know about that.

**Comment by jcupitt** [comment]
Nice, lots of useful fixups.

File: spec/connection_spec.rb (hunk 1 above)
Comment (pftg) [reply spec/connection_spec.rb:3]: just added ;)

**Pull request merged**